pokemongo_bot/step_walker.py

```diff
@@ -20,13 +20,17 @@ def __init__(self, bot, dest_lat, dest_lng):
             dest_lng
         )
 
+        if self.bot.config.walk_min > self.bot.config.walk_max:
+            self.bot.config.walk_min, self.bot.config.walk_max = self.bot.config.walk_max, self.bot.config.walk_min
+            
         self.speed = self.bot.config.walk_max - random() * (self.bot.config.walk_max - self.bot.config.walk_min)
 
         self.destLat = dest_lat
         self.destLng = dest_lng
         self.totalDist = max(1, self.dist)
 
         if self.speed == 0:
+            self.speed = uniform(3,5)
```

Comment:
@TheSavior I see what you mean. I will fix it. Just making sure, by fix I assume fix to remove? According to @douglascamata, we should have it removed asap.